aten/src/ATen/native/Aliases.cpp.in

```diff
@@ -0,0 +1,7 @@
+#include <ATen/ATen.h>
+
+namespace at { namespace native {
+
+$alias_definitions
```

Comment:
Could we see some sample generated code?

Reply:
The template parameter is replaced with:

```
Tensor absolute(const Tensor& self){
	return at::abs(self);
}
Tensor& absolute_(Tensor& self){
	return at::abs_(self);
}
Tensor& absolute_out(const Tensor& self, Tensor& out){
	return at::abs_out(out, self);
}
Tensor arccos(const Tensor& self){
	return at::acos(self);
}
Tensor& arccos_(Tensor& self){
	return at::acos_(self);
}
Tensor& arccos_out(const Tensor& self, Tensor& out){
	return at::acos_out(out, self);
}
Tensor concat(TensorList tensors, int64_t dim){
	return at::cat(tensors, dim);
}
Tensor& concat_out(TensorList tensors, int64_t dim, Tensor& out){
	return at::cat_out(out, tensors, dim);
}
Tensor concat_names(TensorList tensors, Dimname dim){
	return at::cat(tensors, dim);
}
Tensor& concat_names_out(TensorList tensors, Dimname dim, Tensor& out){
	return at::cat_out(out, tensors, dim);
}
Tensor clip(const Tensor& self, const c10::optional<Scalar>& min, const c10::optional<Scalar>& max){
	return at::clamp(self, min, max);
}
Tensor clip_Tensor(const Tensor& self, const c10::optional<Tensor>& min, const c10::optional<Tensor>& max){
	return at::clamp(self, min, max);
}
Tensor& clip_(Tensor& self, const c10::optional<Scalar>& min, const c10::optional<Scalar>& max){
	return at::clamp_(self, min, max);
}
Tensor& clip__Tensor(Tensor& self, const c10::optional<Tensor>& min, const c10::optional<Tensor>& max){
	return at::clamp_(self, min, max);
}
Tensor& clip_out(const Tensor& self, const c10::optional<Scalar>& min, const c10::optional<Scalar>& max, Tensor& out){
	return at::clamp_out(out, self, min, max);
}
Tensor& clip_Tensor_out(const Tensor& self, const c10::optional<Tensor>& min, const c10::optional<Tensor>& max, Tensor& out){
	return at::clamp_out(out, self, min, max);
}
Tensor moveaxis_intlist(const Tensor& self, IntArrayRef source, IntArrayRef destination){
	return at::movedim(self, source, destination);
}
Tensor moveaxis_int(const Tensor& self, int64_t source, int64_t destination){
	return at::movedim(self, source, destination);
}
Tensor& less_Scalar_out(const Tensor& self, const Scalar& other, Tensor& out){
	return at::lt_out(out, self, other);
}
Tensor less_Scalar(const Tensor& self, const Scalar& other){
	return at::lt(self, other);
}
Tensor& less_Tensor_out(const Tensor& self, const Tensor& other, Tensor& out){
	return at::lt_out(out, self, other);
}
Tensor less_Tensor(const Tensor& self, const Tensor& other){
	return at::lt(self, other);
}
Tensor& less__Scalar(Tensor& self, const Scalar& other){
	return self.lt_(other);
}
Tensor& less__Tensor(Tensor& self, const Tensor& other){
	return self.lt_(other);
}
Tensor det(const Tensor& self){
	return at::linalg_det(self);
}
Tensor& det_out(const Tensor& self, Tensor& out){
	return at::linalg_det_out(out, self);
}
```

Reply:
Thanks for the code, this is cool!